minifi-docs/src/main/markdown/System_Admin_Guide.md

```diff
@@ -951,3 +951,22 @@ Provenance Reporting:
     timeout: 30 secs
     batch size: 1000
 ```
+
+# Recommended Antivirus Exclusions
+
+Antivirus software can take a long time to scan directories and the files within them. Additionally, if the antivirus software locks files or directories during a scan, those resources are unavailable to MiNiFi processes, causing latency or unavailability of these resources in a MiNiFi instance. To prevent these performance and reliability issues from occurring, it is highly recommended to configure your antivirus software to skip scans on the following MiNiFi directories:
+
+### MiNiFi Java:
+
+  - content_repository
+  - flowfile_repository
+  - logs
+  - provenance_repository
+  - state
+
+
+### MiNiFi C++:
+
+  - content_repository
+  - flowfile_repository
+  - provenance_repository
```

Comment:
This should be reserved for the MiNiFi C++ codebase/docs.

Reply:
Removed those lines.